src/modules/test/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+project(testfactorylib)
+
+add_library(testfactorylib MimParser.cpp TestRecipeParser.cpp ManagementModule.cpp)

```

Comment:
Same as above, is this (`ManagementModule.cpp`) a duplicate of `src/platform/ManagementModule.cpp` ?